english/alchemy/alchemy.md

```diff
@@ -15,11 +15,10 @@ You decide to play it safe and repeat the process a few times. Who knows how man
 the high-score currently is or how many you will need on your adventure?
 
 The creation process gets quicker and quicker each time you practice the ancient art of
-alchemy. Your skills improve. However, on the fifth run, the coin has an air anomaly
-in its centre.
+alchemy. Your skills improve. However, on the fifth run, the molten metal moves  to the edges of the mold as if blown by air.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.